package.json

```diff
@@ -19,8 +19,8 @@
     "build:types": "tsc --project tsconfig.build.json --module esnext --declarationDir ./dist/types --emitDeclarationOnly --declaration --declarationMap",
     "test": "hardhat test",
     "test:watch": "hardhat watch test",
-    "lint": "eslint --fix src",
-    "lint-check": "eslint src",
+    "lint": "eslint --fix src test",
```

Comment:
side note: do you think we need to lint `e2e` also?

And do you think we should lint the `hardhat.config.ts`?

Reply:
Done